notification_configuration.go

```diff
@@ -150,6 +164,9 @@ type NotificationConfigurationCreateOptions struct {
 
 	// Optional: The list of users belonging to the organization that will receive notification emails.
 	EmailUsers []*User `jsonapi:"relation,users,omitempty"`
+
+	// Required: The workspace or team that the notification configuration is associated with.
+	SubscribableChoice *NotificationConfigurationSubscribableChoice `jsonapi:"polyrelation,subscribable"`
```

Comment:
Do you want to `omitempty` this? That would seem to be typical

Reply:
good catch